resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,70 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
+    	
+    	".*\\..*Id" : "r",
+        ".*\\.id" : "r",
+        ".*\\.data" : "r",
         ".*\\.description" : "cru",
         ".*\\.isPublic" : "cru",
-        ".*\\.kind" : "cr",
+        ".*\\.kind" : "r",
         ".*\\.name" : "cru",
         ".*\\.removeTime" : "ru",
-        ".*\\.uuid" : "cr",
-
+        ".*\\.uuid" : "r",
+        ".*\\.accountId" : "r",
+        ".*\\.created" : "r",
+        ".*\\.state" : "r",
+        ".*\\.removed" : "r",
+        ".*\\.transitioning" : "r",
+        ".*\\.transitioningMessage" : "r",
+        ".*\\.transitioningProgress" : "r",
+        
+        "account" : "crud",
+        "account.id" : "cru",
         "account.externalId" : "cru",
         "account.externalIdType" : "cru",
-        "account.defaultCredentialIds" : "cru",
-        "account.defaultNetworkIds" : "cru",
+        "account.kind" : "cr",
```

Comment:
Yeah, so the "kind" and the "uuid" fields required to be "cr" for some tests, otherwise, they were failing. I need to recall which tests were those. I had a chat with @ibuildthecloud  about this change.